### PR TITLE
 Stacks Using Arrays in c

### LINKED ISSUE #1023: Stacks Using Arrays in C

### PR DESCRIPTION
**Pull request by somya-kapoor**
Fixes #1023

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

-  Added Stacks Using Arrays in c

#### Languages Used:
- C

#### Files Added:
-  Stacks_Using_Arrays.c

### CODE REVIEW

**Comment by somya-kapoor** [comment]
@SakshiShreya I have done the required changes . Please review.

File: Stacks_Using_Arrays/Stacks_Using_Arrays.c [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    else
+        return (stack[top]);
+}
+
+// Function to print the stack array
+void print()
+{
+    int temp = top - 1;
+
+    printf(" Top of stack is : %d \n " , peek(stack));
```
Comment (SakshiShreya) [inline Stacks_Using_Arrays/Stacks_Using_Arrays.c]: Remove space before `,`
Remove space before `:`

File: Stacks_Using_Arrays/Stacks_Using_Arrays.c [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        
+    if(temp > -1);
+        printf( "Other : ");
+    while(temp > -1)
+        printf( "%d  ",stack[temp--]);
```
Comment (SakshiShreya) [inline Stacks_Using_Arrays/Stacks_Using_Arrays.c]: Add space after `,`

File: Stacks_Using_Arrays/Stacks_Using_Arrays.c [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+// Function to print the stack array
+void print()
+{
+    int temp = top - 1;
+
+    printf(" Top of stack is : %d \n " , peek(stack));
+        
+    if(temp > -1);
+        printf( "Other : ");
```
Comment (SakshiShreya) [inline Stacks_Using_Arrays/Stacks_Using_Arrays.c]: Remove space before `:`

File: Stacks_Using_Arrays/Stacks_Using_Arrays.c [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+{
+    int choice, value; 
+    printf(" 1.Push\n ");
+    printf(" 2.Pop\n ");
+    printf(" 3.Display\n ");
+    printf(" 4.Quit\n ");
+    
+    do 
+    {
+        printf("Enter your choice : ");
```
Comment (SakshiShreya) [inline Stacks_Using_Arrays/Stacks_Using_Arrays.c]: Remove space `:`

File: Stacks_Using_Arrays/Stacks_Using_Arrays.c [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+                    
+            case 3: print();
+                    break;
+                    
+            case 4: printf("Exit \n");
+                    break;
+                    
+            default: printf("Invalid Choice \n");            
+        }  
+    } while(choice!=4);
```
Comment (SakshiShreya) [inline Stacks_Using_Arrays/Stacks_Using_Arrays.c]: Add space around `!=`

File: Stacks_Using_Arrays/Stacks_Using_Arrays.c [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    int choice, value; 
+    printf(" 1.Push\n ");
+    printf(" 2.Pop\n ");
+    printf(" 3.Display\n ");
+    printf(" 4.Quit\n ");
+    
+    do 
+    {
+        printf("Enter your choice : ");
+        scanf("%d",&choice);
```
Comment (SakshiShreya) [inline Stacks_Using_Arrays/Stacks_Using_Arrays.c]: Add space after `,`

**Comment by SakshiShreya** [comment]
```
 1.Push
  2.Pop
  3.Display
  4.Quit
 Enter your choice :
```

This is the first thing that the user will see on the console. Please remove the extra space after `\n`

**Comment by somya-kapoor** [comment]
@SakshiShreya I have done the required changes . Please review.

File: Stacks_Using_Arrays/Stacks_Using_Arrays.c [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+        printf("Stack Overflow\n");
+    else
+        stack[++top] = value;
+}
+
+// Function to pop the top value in stack array
+void pop()
+{
+    if(top == -1)
+        printf(" Stack Underflow \n ");
```
Comment (SakshiShreya) [inline Stacks_Using_Arrays/Stacks_Using_Arrays.c]: Fix the extra spacing in this message also. Make it similar to other messages.

File: Stacks_Using_Arrays/Stacks_Using_Arrays.c [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+void print()
+{
+    int temp = top - 1;
+
+    printf("Top of stack is: %d \n", peek(stack));
+        
+    if(temp > -1);
+        printf( "Other: ");
+    while(temp > -1)
+        printf( "%d", stack[temp--]);
```
Comment (SakshiShreya) [inline Stacks_Using_Arrays/Stacks_Using_Arrays.c]: Remove space after `(` in lines 45 and 47.

File: Stacks_Using_Arrays/Stacks_Using_Arrays.c [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+    printf( "\n" );
```
Comment (SakshiShreya) [inline Stacks_Using_Arrays/Stacks_Using_Arrays.c]: Remove extra space before and after the string.

**Comment by somya-kapoor** [comment]
@SakshiShreya I have done the required changes . Please review.

**Pull request merged**